### ISSUE TITLE
Format use for return date

### ISSUE THREAD

**Issue by hehrhart** [Supermood/dev-handbook#4]
__QUESTIONS__: For createdAt and updatedAt in the function toArray (), there is not always the same output format. I find one format un [AutomaticConfig.php](https://github.com/Supermood/main-app/blob/master/supermood/platform/backend/services/supermood-api/app/User/Rights/GlobalRights/ResultsViewer/AutomaticConfig/AutomaticConfig.php#L48) and other one in [User.php](https://github.com/Supermood/main-app/blob/master/supermood/platform/backend/services/supermood-api/app/User.php#L111).

__QUESTIONS__:  We use this format in all returns or only in the `toArray ()` function included in the models ?

**Comment by hilnius** [comment]
Shouldn't we use ISO 8601 ? 
which means we should use ```$dateObject->format('c')``` ?

**Comment by hehrhart** [comment]
Only if you want, no need to use iso, you need to choose one format and class for construct. And add section on handbook for explain how create and passing info.

**Comment by slashome** [comment]
@hilnius I agree with you, I just think we should format with the dedicated method for a better visibility 
```php
Carbon::instance($this->date)->toIso8601String()
```
